src/app/components/universal-tabbed-card/universal-tab/universal-tab.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-universal-tab',
+  templateUrl: './universal-tab.component.html',
+  styleUrls: ['./universal-tab.component.scss']
+})
+export class UniversalTabComponent {
+
+  @Input() tabTitle: string = 'Tab';
```

Comment:
FTFY

Reply:
Not sure if it'd be better to leave it blank, but in case something goes wrong, this would be the default value. So the tab would be called 'Tab' instead of just nothing.